lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -1434,14 +1434,30 @@ def show_favorites # :nodoc:
               return
             end
 
-            # create module set using the saved modules
-            fav_modules = {}
-            saved_favs = File.readlines(favs_file)
-            saved_favs.each do |mod|
-              module_name = mod.strip
-              fav_modules[module_name] = framework.modules[module_name]
-            end
-            show_module_metadata('Favorites', fav_modules)
+            # get the full module names from the favorites file and use then to search the MetaData Cache for matching modules
+            saved_favs = File.readlines(favs_file).map(&:strip)
```

Comment:
Added this code back in to populate our hash to generated our table via `show_module_metadata`.
```suggestion
            saved_favs = File.readlines(favs_file).map(&:strip)
            saved_favs.each do |mod|
              # populate hash with module fullname and module object
              fav_modules[mod] = framework.modules[mod]
            end
```